ignite/handlers/lr_finder.py

```diff
@@ -387,9 +391,10 @@ def attach(
                 iteration. It must return the loss of that iteration.
             num_iter: number of iterations for lr schedule between base lr and end_lr. Default, it will
                 run for ``trainer.state.epoch_length * trainer.state.max_epochs``.
+            start_lr: lower bound for lr search. Default, Learning Rate specified with the optimizer.
```

Comment:
Not related to this line. Could you please add to `to_save` docstring above and mention that it should contain optmizer by the key "optimizer". For example,
```
It should contain "optimizer" key for the optimizer.
```

Reply:
Done.